perfkitbenchmarker/linux_virtual_machine.py

```diff
@@ -257,8 +261,19 @@ def RemoteCommand(self, command,
                     should_log=False, retries=SSH_RETRIES,
                     ignore_failure=False, login_shell=False,
                     suppress_warning=False):
+    return self.RemoteHostCommand(command, should_log, retries,
+                                  ignore_failure, login_shell,
+                                  suppress_warning)
+
+  def RemoteHostCommand(self, command,
```

Comment:
ContainerizedDebianMixin.RemoteHostCopy can just call super(ContainerizedDebianMixin, self).RemoteCopy(...)   - same for remotehostcommand and movehostfile - there wouldn't be too much duplicate code. And I do feel that adding Host methods to the linuxmixin complicates things unnecessarily